<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::CreateProjectDialog</class>
 <widget class="QDialog" name="tt3::gui::CreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Actions/CreateProjectLarge.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreateProjectLarge.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="parentProjectLabel">
     <property name="text">
      <string>Parent:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="parentProjectComboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="selectParentProjectPushButton">
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-gui.qrc">
       <normaloff>:/tt3-gui/Resources/Images/Actions/ViewProjectSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ViewProjectSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="displayNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="displayNameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="descriptionPlainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>85</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="beneficiariesLabel">
     <property name="text">
      <string>Beneficiaries:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="beneficiariesValueLabel">
     <property name="text">
      <string>-</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="selectBeneficiariesPushButton">
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-gui.qrc">
       <normaloff>:/tt3-gui/Resources/Images/Actions/ViewBeneficiarySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ViewBeneficiarySmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="completedCheckBox">
     <property name="text">
      <string>Project is completed</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>parentProjectComboBox</tabstop>
  <tabstop>selectParentProjectPushButton</tabstop>
  <tabstop>displayNameLineEdit</tabstop>
  <tabstop>descriptionPlainTextEdit</tabstop>
  <tabstop>selectBeneficiariesPushButton</tabstop>
  <tabstop>completedCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectParentProjectPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>_selectParentProjectPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>_displayNameLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionPlainTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>_descriptionPlainTextEditTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectBeneficiariesPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::CreateProjectDialog</receiver>
   <slot>_selectBeneficiariesPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_selectParentProjectPushButtonClicked()</slot>
  <slot>_displayNameLineEditTextChanged(QString)</slot>
  <slot>_descriptionPlainTextEditTextChanged()</slot>
  <slot>_selectBeneficiariesPushButtonClicked()</slot>
 </slots>
</ui>
